code/modules/surgery/organs/vocal_cords.dm

```diff
@@ -396,18 +396,20 @@ GLOBAL_DATUM_INIT(multispin_words, /regex, regex("like a record baby"))
 	else if((findtext(message, GLOB.rest_words)))
 		for(var/V in listeners)
 			var/mob/living/L = V
-			if(!L.resting)
+			if(!IS_HORIZONTAL(L))
 				L.lay_down()
 		next_command = world.time + cooldown_meme
 
 	//GET UP
 	else if((findtext(message, GLOB.getup_words)))
 		for(var/V in listeners)
 			var/mob/living/L = V
-			if(L.resting)
-				L.lay_down() //aka get up
+			if(IS_HORIZONTAL(L))
```

Comment:
should this if statement be after the setting of other things to 0?

Reply:
uh no, removing the weakened, etc will make them stand up anyway.

Reply:
it won't do it if they aren't weakened/floored etc. 
so if they are just lying down, this is needed.